clisops/core/subset.py

```diff
@@ -1171,15 +1154,6 @@ def subset_shape(
     else:
         ds_copy = ds_copy.where(inner_mask, drop=True)
 
-    # Add a CRS definition using CF conventions and as a global attribute in CRS_WKT for reference purposes
-    ds_copy.attrs["crs"] = raster_crs.to_string()
-    ds_copy["crs"] = 1
-    ds_copy["crs"].attrs.update(raster_crs.to_cf())
-
-    for v in ds_copy.variables:
-        if {lat.name, lon.name}.issubset(set(ds_copy[v].dims)):
-            ds_copy[v].attrs["grid_mapping"] = "crs"
-
```

Comment:
If the issue has to do with a data variable called `crs`, what are the alternative approaches? Would a coordinate variable named `crs` make more sense?

Reply:
a coordinate variable named `crs` was my suggestion at first. But Pascal pointed out that the crs guessing did not make sense in some cases with rotated_pole.

I think I removed too much here though, trying to fix it now.